xudo/commands/build.py

```diff
@@ -1,8 +1,6 @@
 """The logs command."""
 
 
-from json import dumps
```

Comment:
I didn't see dumps being used, LMK if its needed for stdout and I can add it back in